includes/class-algolia-settings.php

```diff
@@ -92,31 +92,42 @@ public function get_api_key() {
 	}
 
 	/**
-	 * Get the post types blacklist.
+	 * Get the excluded post types.
 	 *
 	 * @author  WebDevStudios <contact@webdevstudios.com>
 	 * @since   1.0.0
 	 *
 	 * @return array
 	 */
-	public function get_post_types_blacklist() {
-		$blacklist = (array) apply_filters( 'algolia_post_types_blacklist', array( 'nav_menu_item' ) );
+	public function get_excluded_post_types() {
```

Comment:
Since this is "public" code, i.e., it could be extended, and we don't know how others might be currently using it, I would prefer creating new methods, rather than just renaming the originals.

In the original methods that are being deprecated, we can throw a deprecation notice, and then return the result of their replacement methods.

Example (might need some work, just off the cuff)...
```
/**
 * Get the excluded post types.
 *
 * @author     WebDevStudios <contact@webdevstudios.com>
 * @since      1.0.0
 * @deprecated 1.7.0 Use Algolia_Settings::get_excluded_post_types()
 * @see        Algolia_Settings::get_excluded_post_types()
 *
 * @return array
 */
public function get_post_types_blacklist() {
	_deprecated_function(
		__METHOD__,
		'1.7.0',
		'Algolia_Settings::get_excluded_post_types();'
	);
	return $this->get_excluded_post_types();
}

/**
 * Get the excluded post types.
 *
 * @author WebDevStudios <contact@webdevstudios.com>
 * @since  1.7.0
 *
 * @return array
 */
public function get_excluded_post_types() {
	// New method code here...
}
```